Podcast/UserDetailViewController.swift

```diff
@@ -0,0 +1,413 @@
+//
+//  UserDetailViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 3/7/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+final class UserDetailViewController: ViewController {
+    
+    let episodeCellReuseId = "EpisodeCell"
+    let nullEpisodeCellReuseId = "NullEpisodeCell"
+    let seriesCellReuseId = "SeriesCell"
+    let nullSeriesCellReuseId = "NullSeriesCell"
+    
+    let headerViewReuseId = "headerViewId"
+    
+    let recastsHeaderViewTag = 2112
+    let subscriptionsHeaderViewTag = 3110
+    
+    var scrollYOffset: CGFloat = 118
+    
+    var profileTableView: UITableView!
+    var userDetailHeaderView: UserDetailHeaderView!
+    var navBar: UserDetailNavigationBar!
+    var isLoading: Bool = true
+    
+    var user: User!
+    var subscriptions: [Series] = []
```

Comment:
I think these should be optional .. that way we know the difference between loading and no subscriptions / recasts

Reply:
Hmm it was like that before and it was a pain in the ***. This way is more swift-y with no nulls.